sagemaker/24_train_bloom_peft_lora/sagemaker-notebook.ipynb

```diff
@@ -428,24 +444,572 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
     "predictor.delete_model()\n",
     "predictor.delete_endpoint()"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
+  "availableInstances": [
```

Comment:
Thats should not be there.

Reply:
Sorry, sloppy on my part. Will remove on next commit.

Reply:
Spent like 2 hours trying to remove all this extra metadata SM Studio adds, I ended up just opening a superseding PR #373 to address. Separately reaching out to SM Studio team for workaround / remove this going forward.